scipy/interpolate/_fitpack_py.py

```diff
@@ -564,6 +567,66 @@ def spalde(x, tck):
     .. [3] Dierckx P. : Curve and surface fitting with splines, Monographs on
        Numerical Analysis, Oxford University Press, 1993.
 
+    Examples
+    --------
+    To calculate the derivatives of a B-spline there are several aproaches. 
+    In this example, we will demonstrate that `spalde` is equivalent to
+    calling `splev` and `splder`.
+    
+    >>> import numpy as np
+    >>> import matplotlib.pyplot as plt
+    >>> from scipy.interpolate import BSpline, spalde, splder, splev
+    
+    >>> # Store characteristic parameters of a B-spline
+    >>> # (knots, coefficients, degree of the spline)
+    >>> tck = ((-2, -2, -2, -2, -1, 0, 1, 2, 2, 2, 2),
+    ...        (0, 0, 0, 6, 0, 0, 0),
+    ...        3)
+    >>> # Instance a B-spline object
+    >>> # `BSpline` objects are prefered, except for spalde()
+    >>> bspl = BSpline(tck[0], tck[1], tck[2])
+    >>> # Generate extra points to get a smooth curve
+    >>> x = np.linspace(min(tck[0]), max(tck[0]), 100)
+    
+    Evaluate the curve and all derivatives
+    
+    >>> # The order of derivative must be less or equal to k, the degree of the spline
+    >>> # Method 1: splev()
+    >>> m1_y_bsplin = splev(x, bspl, der=0)  # The B-spline itself
+    >>> m1_y_deriv1 = splev(x, bspl, der=1)  # 1st derivative
+    >>> m1_y_deriv2 = splev(x, bspl, der=2)  # 2nd derivative
+    >>> m1_y_deriv3 = splev(x, bspl, der=3)  # 3rd derivative
+    >>> # Method 2: splder()
+    >>> m2_y_bsplin = splder(bspl, n=0)(x)
+    >>> m2_y_deriv1 = splder(bspl, n=1)(x)
+    >>> m2_y_deriv2 = splder(bspl, n=2)(x)
+    >>> m2_y_deriv3 = splder(bspl, n=3)(x)
+    >>> # Method 3: spalde()
+    >>> m3_y_bsplin = [spalde(i, tck)[0] for i in x ]
+    >>> m3_y_deriv1 = [spalde(i, tck)[1] for i in x ]
+    >>> m3_y_deriv2 = [spalde(i, tck)[2] for i in x ]
+    >>> m3_y_deriv3 = [spalde(i, tck)[3] for i in x ]
+    
+    Graphic comparison
+    
+    >>> # Generate a figure with three axes for graphic comparison
```

Comment:
This sentence is not needed really, given that you actually generate said figure just below.